pkg/app/impl/authz.go

```diff
@@ -55,10 +58,18 @@ func NewAuthorizerServer(
 ) *AuthorizerServer {
 	newLogger := logger.With().Str("component", "api.grpc").Logger()
 
+	const certs = `https://citadel.demo.aserto.com/dex/keys`
+
+	// TODO: get context
+	ctx := context.Background()
+	jwtCache := jwk.NewCache(ctx)
+
 	return &AuthorizerServer{
 		cfg:      cfg,
 		logger:   &newLogger,
 		resolver: rf,
+		issuers:  make(map[string]string),
```

Comment:
Should this be a `sync.Map` as there will be concurrent RW access from various handlers?